packages/assets-controllers/src/TokenRatesController.ts

```diff
@@ -322,18 +322,10 @@ export class TokenRatesController extends PollingControllerV1<
    * Updates exchange rates for all tokens.
    */
   async updateExchangeRates() {
-    if (this.tokenList.length === 0 || this.disabled) {
-      return;
-    }
-
     const { chainId, nativeCurrency } = this.config;
-    const tokenContractAddresses = this.tokenList.map(
-      (token) => token.address,
-    ) as Hex[];
```

Comment:
Here is the `as Hex` call that was the reason we didn't need `toHex` prior to now